src/components/NanoContract/SelectAddressModal.js

```diff
@@ -115,7 +118,6 @@ export const SelectAddressModal = ({ address, show, onSelectAddress, onDismiss }
                   <Text style={styles.infoText}>{t`To change, select other address on the list below.`}</Text>
                   <Text>
                     <TextValue bold>{t`Address`}</TextValue>
-                    {/* the unicode character u00A0 means no-break space. */}
```

Comment:
suggestion: I think it's best to keep this comment, as it's very self explanatory.